### ISSUE TITLE
Bug report: - can not save cloned theme

### ISSUE THREAD

**Issue by Owyn** [Nexus-Mods/Vortex#5797]
**Describe the bug**
can not save cloned theme

**To Reproduce**
1. Go to theme settings
2. Click on clone it
3. Try saving

**Expected behavior**
No error

**Screenshots**
![изображение](https://user-images.githubusercontent.com/1309656/70243502-bc911200-1783-11ea-8e6f-c21b2db9a401.png)

**Platform (please complete the following information):**
 - OS: Windows 10
 - Vortex Version 1.1.4

**Additional context**
details:
```
ENOENT: no such file or directory, open 'S:\Users\owyn\AppData\Roaming\Vortex\themes\S:\Users\owyn\AppData\Roaming\Vortex\themes\default_my\variables.scss'
Error
    at Object.args [as writeFileAsync] (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\renderer.js:125:182)
    at Object.saveThemeInternal (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\theme-switcher\index.js:1:30158)
    at Object.saveTheme.e [as onApply] (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\theme-switcher\index.js:1:25911)
    at apply (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins\theme-switcher\index.js:1:33919)
    at Object.ca (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:14:428)
    at ja (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:15:118)
    at ka (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:15:172)
    at wa (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:17:210)
    at Aa (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:18:248)
    at ya (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:18:57)
    at Da (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:21:69)
    at Ad (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:84:409)
    at Gi (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:250:397)
    at Kb (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:39:140)
    at Dd (S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:86:249)
    at S:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar\node_modules\react-dom\cjs\react-dom.production.min.js:251:229
```

**Comment by urielz** [comment]
can reproduce this as well

**Comment by TanninOne** [comment]
This was fixed in 1.1.5
